docs/macos_cheatsheet.md

```diff
@@ -2,10 +2,10 @@ Hello and welcome to this Apple M1 Cheatsheet.
 
 The goal is to gather here all tips and known issues about the new processor from Apple!
 
-- [Issues](https://github.com/lewagon/setup/blob/master/apple_m1_cheatsheet.md#issues)
-  - [Install brew](https://github.com/lewagon/setup/blob/master/apple_m1_cheatsheet.md#install-brew)
-  - [ZSH Permissions](https://github.com/lewagon/setup/blob/master/apple_m1_cheatsheet.md#zsh-permissions)
-- [Tips](https://github.com/lewagon/setup/blob/master/apple_m1_cheatsheet.md#tips)
+- [Issues](https://github.com/lewagon/setup/blob/master/docs/macos_cheatsheet.md#issues)
+  - [Install brew](https://github.com/lewagon/setup/blob/master/docs/macos_cheatsheet.md#install-brew)
+  - [ZSH Permissions](https://github.com/lewagon/setup/blob/master/docs/macos_cheatsheet.md#zsh-permissions)
```

Comment:
For this here, it doesn't seem to have any info associated with it. I just left it in in case the info that went with it is found. Otherwise, that line can be removed since it would be pointing to non-existent information

Reply:
With [this option ](https://github.com/lewagon/dotfiles/blob/f30d1b4776d1f020dbe672ff7c39f5e0a253f659/zshrc#L13) recently added to the dotfiles, there is no zsh permission issue anymore. We can remove the line.